docs/docs/developers/deployment/aws/setup-guide/index.md

```diff
@@ -1 +1,200 @@
+    deploy:
+      resources:
+        limits:
+          cpus: '1'
+          memory: 4096M
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=mongoadmin
+      - MONGO_INITDB_ROOT_PASSWORD_FILE=/run/secrets/mongo.password
+    volumes:
+      - mongo_repository:/data/db
+    secrets:
+      - source: mongo_password
+        target: mongo.password
+```
+8. Change `nginx` service definition to:
+```yaml
+  nginx:
+    image: <NGINX_ECR_IMAGE_URI>
+    ports:
+      - target: 80
+        published: 80
+        x-aws-protocol: http
+    volumes:
+      - site_repository:/usr/share/nginx/html:ro
+```
+    - `NGINX_ECR_IMAGE_URI` - replace with Nignx repository URI from [AWS ECR dashboard](https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-info.html)
+    - `x-aws-protocol` - configuration for AWS ALB, read more [here](https://docs.docker.com/cloud/ecs-compose-examples/#service)
+9. Add `x-aws-cloudformation` section with Load Balancer configuration:
+```yaml
+x-aws-cloudformation:
+  Resources:
+    Cms8080TargetGroup:
+      Properties:
+        HealthCheckProtocol: HTTP
+        HealthCheckPort: 8080
+        HealthCheckPath: /system/health
+        Matcher:
+          HttpCode: 200
+    Nginx80TargetGroup:
+      Properties:
+        HealthCheckProtocol: HTTP
+        HealthCheckPort: 80
+        HealthCheckPath: /health
+        Matcher:
+          HttpCode: 200
+    Cms8080Listener:
+      Properties:
+        Certificates:
+          - CertificateArn: "<CERTIFICATE_ARN>"
+        Protocol: HTTPS
+        Port: 8443
+    Nginx80Listener:
+      Properties:
+        Certificates:
+          - CertificateArn: "<CERTIFICATE_ARN>"
+        Protocol: HTTPS
+        Port: 443
+```
+    - `CERTIFICATE_ARN` - replace with ARN of the certificate generated in [AWS Configuration](#step-1-aws-configuration) step.
+10. [Create AWS ECS context](https://docs.docker.com/cloud/ecs-integration/#create-aws-context) named `ws-ecs`:
+    - `docker context create ecs ws-ecs`
+11. Update `distribution/src/main/docker/nginx/default.conf` Nginx config with additional health endpoint:
+```apache
+    location /health {
+        access_log off;
+        add_header  Content-Type    text/html;
+        return 200;
+    }
+```
+
+To find more information about using Docker Compose with AWS Elastic Container Service, please read [Deploying Docker containers on ECS](https://docs.docker.com/cloud/ecs-integration/).
+
+### Maven
+
+1. Update `io.fabric8:docker-maven-plugin` plugin configuration in `distribution/pom.xml`. 
+    - Add the following `buildx` extension to cms and nginx images `<build>` sections:
+```xml
+                <buildx>
+                  <platforms>
+                    <platform>linux/amd64</platform>
+                    <platform>linux/arm64</platform>
+                  </platforms>
+                </buildx>
+```
+    - Add the following `execution`:
+```xml
+          <execution>
+            <id>push-docker-image</id>
```

Comment:
how this will affect  regular builds without need of pushing to ECR?

Reply:
Yeah, I though about it too yesterday.
The most "clean" solution is the one we have in https://github.com/websight-io/starter (Maven profiles). However, it would make this tutorial unnecessarily complicated. I think a simple `skip` property should be better in this case. Thanks for pointing out.